packages/fluentui/react-northstar-prototypes/src/prototypes/FormValidation/index.tsx

```diff
@@ -1,19 +1,10 @@
 import * as React from 'react';
 import { PrototypeSection, ComponentPrototype } from '../Prototypes';
-import FormValidateFormik from './Formik';
```

Comment:
Is there still supposed to be stuff removed from this file?

Reply:
Hmm, this may have been a bad merge; i'll take a look.

Reply:
fixed